src/components/App.tsx

```diff
@@ -1393,9 +1380,11 @@ class App extends React.Component<ExcalidrawProps, AppState> {
               return;
             case SCENE.INIT: {
               if (!this.portal.socketInitialized) {
-                this.updateScene(decryptedData.payload.elements, {
-                  init: true,
-                });
+                const remoteElements = decryptedData.payload.elements;
+                history.resumeRecording();
+                this.setScrollToCenter(remoteElements);
+                this.updateScene(remoteElements);
+                this.initializeSocket();
```

Comment:
Is there be any case during `SCENE.UPDATE` whenn socket is still not initialized?